src/sql/sqlops.proposed.d.ts

```diff
@@ -715,7 +715,7 @@ declare module 'sqlops' {
 				 * Register model view content for the dialog.
 				 * Doesn't do anything if model view is already registered
 				 */
-				registerContent(handler: (view: ModelView) => void): void;
+				registerContent(handler: (view: ModelView) => Thenable<void>): void;
```

Comment:
is this a "breaking change" that we need to communicate with partners?

Reply:
@kburtram I don't think it is but I'll do that